prime-router/src/testIntegration/kotlin/transport/RESTTransportIntegrationTests.kt

```diff
@@ -641,16 +641,15 @@ hnm8COa8Kr+bnTqzScpQuOfujHcFEtfcYUGfSS6HusxidwXx+lYi1A==
     fun `test  transport postReport with valid file name for Natus`() {
         val header = makeHeader()
         val mockRestTransport = spyk(RESTTransport(mockClientStringTokenOk()))
-
         // Given:
         //      lookupDefaultCredential returns mock UserPassCredential object to allow
         //      the getAuthTokenWithUserPass() to be called.
         //      expectedFileName is file name to send to NATUS.
         val formatter = DateTimeFormatter.ofPattern("yyyyMMddHHmmss")
-        val expectedFileName = "ignore-${header.reportFile.reportId}-" +
-            "${formatter.format(header.reportFile.createdAt)}.hl7"
+        val expectedFileName = "${header.reportFile.schemaName}-${header.reportFile.reportId}-" +
```

Comment:
@victor-chaparro and @oslynn This test assumes blobURL is going to be empty, if that is the case, this is the new format of the file name. In practice, I think blobURL will always be populated, so we will pull the name from there. Is that okay? IMO that is better than what we have now.